nix/builders/nanopc-t6/base.nix

```diff
@@ -70,14 +70,30 @@
     };
 
   environment.systemPackages = with pkgs; [
+    avahi
     cloud-utils
     parted
     wpa_supplicant
     screen
   ];
 
+  services.avahi = {
+      nssmdns4 = true;
+      nssmdns6 = true;
+
+      enable = true;
+      reflector = true;
+      hostName = "dogebox";
```

Comment:
During initial setup we get the user to select a hostname for their box that is then set. If this stays as `dogebox` this might confuse the user, especially if they have multiple boxes on their network. Maybe we should try and read `/etc/hostname` and fall back?